### ISSUE TITLE
Missing padding in footer

### ISSUE THREAD

**Issue by jakubpawlina** [fossasia/gci18.fossasia.org#217]
**Actual Behaviour**

The content in footer is aligned to right and left without any padding.

**Expected Behaviour**

Space between content and edge.

**Steps to reproduce it**

Content contained in div with correct padding.

**LogCat for the issue**

\-

**Screenshots of the issue**

![zrzut ekranu 69](https://user-images.githubusercontent.com/29257057/47967932-725ece00-e063-11e8-9457-03b86f21a839.png)

**Would you like to work on the issue?**

Yes

**Comment by parammittal16** [comment]
@jakubpawlina Can I work on this?

**Comment by jakubpawlina** [comment]
@parammittal16 Ok, you can do it. I can support you if needed :)

**Comment by parammittal16** [comment]
@jakubpawlina I figured out that a div having class container is closing at the wrong place, That's why icons footer elements are going out of the container.
I closed the div in the right position and got the results right. Even there were comments stating to close the container div at last but it was closed above. I corrected it.
![capture01](https://user-images.githubusercontent.com/31361983/48023388-ab7e6780-e163-11e8-8eec-badf31484968.PNG)
![capture02](https://user-images.githubusercontent.com/31361983/48023390-ac16fe00-e163-11e8-8d62-e4755bd3750a.PNG)

Now shall I make the PR?
